lib/factory.js

```diff
@@ -13,9 +14,9 @@ function patchBaseClass(cls) {
   var proto = cls.prototype;
   var found = false;
   // Find the prototype that owns the _setProperties method
-  while (proto
-    && proto.constructor !== pkgcloud.storage.Container
-    && proto.constructor !== pkgcloud.storage.File) {
+  while (proto &&
+    proto.constructor !== pkgcloud.storage.Container &&
+    proto.constructor !== pkgcloud.storage.File) {
```

Comment:
Indent one more level for readability https://github.com/strongloop-internal/loopback-knowledge-base/blob/4772f51fcc68c7c1a83157b31eac5379e509d238/style-guide.md#good-3